cmd/insprd/main.go

```diff
@@ -8,26 +8,36 @@ import (
 	"github.com/inspr/inspr/cmd/insprd/memory/tree"
 	"github.com/inspr/inspr/cmd/insprd/operators"
 	kafka "github.com/inspr/inspr/cmd/insprd/operators/kafka"
+	"github.com/inspr/inspr/pkg/auth"
+	jwtauth "github.com/inspr/inspr/pkg/auth/jwt"
 )
 
 func main() {
 	var memoryManager memory.Manager
 	var operator operators.OperatorInterface
-
+	var authenticator auth.Auth
 	var err error
+
+	pubKey, err := auth.GetPublicKey()
+	if err != nil {
+		panic(err)
+	}
+
 	if _, ok := os.LookupEnv("DEBUG"); ok {
+		authenticator = jwtauth.NewJWTauth(pubKey)
 		memoryManager = tree.GetTreeMemory()
 		operator, err = kafka.NewKafkaOperator(memoryManager)
 		if err != nil {
 			panic(err)
 		}
 	} else {
+		authenticator = jwtauth.NewJWTauth(pubKey)
 		memoryManager = tree.GetTreeMemory()
 		operator, err = kafka.NewKafkaOperator(memoryManager)
 		if err != nil {
 			panic(err)
 		}
```

Comment:
@ptcar2009 @nicholaspcr do you guys know why the code in "if" is the same as in "else"?